src/lib/functionBodyParser.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ *
+ * Convert function to a string ang gets body expressions
+ * this expressions' strings will then be inserted in the worker blob
+ *
+ * @param {Function} [] A function whose will be extracted
+ * @returns {String} The function's body
+ *
+ * @example
+ * functionBodyParser(function () {let a = 2}) // return let a = 2)
+ */
+const functionBodyParser = (fn?: Function) => {
+  const functionString = fn?.toString()
```

Comment:
We could put a condition to check if the function exists. In this way we don't return `undefined`

```javascript
if (!fn) return ''

const functionString = fn.toString()
return functionString.slice(
    functionString.indexOf('{') + 1,
    functionString.lastIndexOf('}')
)
```